common/src/config.rs

```diff
@@ -285,7 +289,9 @@ impl Default for Limits {
         Limits {
             max_concurrent_requests: 100,
             max_concurrent_queries: 5,
-            max_event_payload_size: ByteSize::from_kilobytes(256),
+            max_plain_event_payload_size: ByteSize::from_kilobytes(256),
+            max_security_event_payload_size: ByteSize::from_kilobytes(256),
+            max_minidump_event_payload_size: ByteSize::from_kilobytes(256),
```

Comment:
Must be north of 1GB imo. We can lower the limit on the load balancer if we fancy to.

Reply:
will set it to 2 Gb by default.